elixir/config/test.exs

```diff
@@ -68,6 +68,10 @@ config :api, API.Endpoint,
   url: [port: 13_101],
   server: true
 
+config :api,
+  # shorten debounce timeout for tests
+  relays_presence_debounce_timeout: 100
```

Comment:
Given the lack of types, I think this variable name should say what it is:

```suggestion
  relays_presence_debounce_timeout_ms: 100
```

Reply:
We're only debouncing for 100 milliseconds?

Reply:
@thomaseizinger done